xmrstak/misc/executor.cpp

```diff
@@ -1036,9 +1037,26 @@ void executor::http_hashrate_report(std::string& out)
 	out.append(buffer);
 
 	double fTotal[3] = { 0.0, 0.0, 0.0};
+	auto bTypePrev = static_cast<xmrstak::iBackend::BackendType>(0);
+	std::string name;
+	size_t j = 0;
 	for(size_t i=0; i < nthd; i++)
 	{
 		double fHps[3];
+		char csThreadTag[25];
+		auto bType = static_cast<xmrstak::iBackend::BackendType>(pvThreads->at(i)->backendType);
+		if(bTypePrev == bType){
```

Comment:
`{` must be on  a new line. Independent of the project where you provide code always look how the code style in the file looks like and try to follow this style (even if there is no style guide).

Reply:
repaired